### ISSUE TITLE
Add Repo Labels

### ISSUE THREAD

**Issue by mgifford** [w3c/wai-arrm#63]
We can customize the labels that allow us to bucket these issues better:
https://github.com/w3c/wai-arrm/labels

Things like:
 - WCAG 2.2
 - Document structure
 - Role Definitions

**Comment by mgifford** [comment]
@khawk27 this is both documentation and internal project management. It was focused on labels, but doesn't have to stay there. That's just the easiest way to clump together tasks.
